orca-deploymentmonitor/src/main/java/com/netflix/spinnaker/orca/deploymentmonitor/models/MonitoredDeployInternalStageData.java

```diff
@@ -30,6 +30,7 @@
   private String cloudProvider;
   private int currentProgress;
   private Map<String, Object> parameters;
+  private boolean isFailure;
```

Comment:
`isFailure` is probably a bit too generic and might be confusing for someone reading the execution JSON.
I would rename to something like `hasDeployementFailed`